app/src/main/java/com/summer/practice/tvtracker/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package com.summer.practice.tvtracker
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.os.Handler
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        supportActionBar?.hide();
+
+        Handler().postDelayed({
```

Comment:
please create another activity called SplashActivity and make it main launcher in manifest. Move the splash screen logic from MainActivity to SplashActivity. Also, remember, MainActivity is our home screen, you shouldn't touch it yet